3604/1/top_gems.rb

```diff
@@ -0,0 +1,27 @@
+require './array_names_gems_creator.rb'
+require './gem_info_fetcher.rb'
+require './rating_creator.rb'
+require './table_output.rb'
+require './options_creator.rb'
+
+# class for main task: start project
+class TopGems
+  def start_top_gems
+    hash_options = OptionsCreator.new.hash_options
+    array_name_gems = ArrayNamesGemsCreator.new.array_names_of_gems(hash_options)
+    array_of_all_gems = create_array_with_all_information_about_gems(array_name_gems)
+    array_of_all_gems = RatingCreator.new.array_of_rating_gems(array_of_all_gems)
```

Comment:
one more naming issue. you chose the same name as on previous line. but RatingCreator reorders array of gems by their rating. please reflect this in variable's name and method's name (for example, reordered_gems and reoder_gems_by_rating)